src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallControllerTest.kt

```diff
@@ -103,13 +101,77 @@ class RecallControllerTest {
+  private val inAssessmentRecall = newRecall().copy(bookedByUserId = bookedByUserId.value, assignee = assignee.value)
+  private val stoppedRecall = newRecall().copy(bookedByUserId = bookedByUserId.value, agreeWithRecall = AgreeWithRecall.NO_STOP)
+  private val recallNotificationIssuedRecall = newRecall().copy(recallNotificationEmailSentDateTime = now)
+  private val dossierInProgressRecall = newRecall().copy(recallNotificationEmailSentDateTime = now, assignee = assignee.value)
+  private val dossierIssuedRecall = newRecall().copy(dossierCreatedByUserId = dossierCreatedByUserId.value)
+
+  @Test
+  fun `gets all recalls for a band FOUR_PLUS returns all recalls`() {
+    val bearerToken = "Bearer header.payload"
+    every { recallRepository.findAll() } returns listOf(beingBookedOnRecall, bookedOnRecall, inAssessmentRecall, stoppedRecall, recallNotificationIssuedRecall, dossierInProgressRecall, dossierIssuedRecall)
+    every { tokenExtractor.getTokenFromHeader(bearerToken) } returns Token(::UserId.random().toString())
+    val userDetails = mockk<UserDetails>()
+    every { userDetailsService.get(any()) } returns userDetails
+    every { userDetailsService.find(any()) } returns userDetails
+    every { userDetails.caseworkerBand } returns CaseworkerBand.FOUR_PLUS
+    every { userDetails.fullName() } returns "Mickey Mouse"
+
+    val results = underTest.findAll(bearerToken)
+
+    assertThat(results.size, equalTo(7))
+
+    assertThat(
+      results,
+      List<RecallResponse>::containsAll,
+      listOf(
+        recallResponse(beingBookedOnRecall, Status.BEING_BOOKED_ON),
+        recallResponse(bookedOnRecall, Status.BOOKED_ON).copy(bookedByUserId = bookedByUserId),
+        recallResponse(inAssessmentRecall, Status.IN_ASSESSMENT).copy(bookedByUserId = bookedByUserId, assignee = assignee, assigneeUserName = "Mickey Mouse"),
+        recallResponse(stoppedRecall, Status.STOPPED).copy(bookedByUserId = bookedByUserId, agreeWithRecall = AgreeWithRecall.NO_STOP),
+        recallResponse(recallNotificationIssuedRecall, Status.RECALL_NOTIFICATION_ISSUED).copy(recallNotificationEmailSentDateTime = now),
+        recallResponse(dossierInProgressRecall, Status.DOSSIER_IN_PROGRESS).copy(recallNotificationEmailSentDateTime = now, assignee = assignee, assigneeUserName = "Mickey Mouse"),
+        recallResponse(dossierIssuedRecall, Status.DOSSIER_ISSUED).copy(dossierCreatedByUserId = dossierCreatedByUserId)
+      )
+    )
+  }
+
   @Test
-  fun `gets all recalls`() {
-    every { recallRepository.findAll() } returns listOf(recall)
+  fun `gets all recalls for a band THREE returns only recalls in BEING_BOOKED_ON, STOPPED, RECALL_NOTIFICATION_ISSUED, DOSSIER_IN_PROGRESS, DOSSIER_ISSUED statuses`() {
+    val bearerToken = "Bearer header.payload"
+    every { recallRepository.findAll() } returns listOf(beingBookedOnRecall, bookedOnRecall, inAssessmentRecall, stoppedRecall, recallNotificationIssuedRecall, dossierInProgressRecall, dossierIssuedRecall)
```

Comment:
No need now but I think I'd prob create a val list `allRecallStates` for this `listOf` and re-use across the two tests.  Also prob the other unchanged stubbing too.  All just to emphasise that the diff is just from the CW band.